src/PainterWidget.py

```diff
@@ -28,6 +34,10 @@ def __init__(self, painter_widget):
         self.last_window = None
 
     def run(self) -> None:
+        if "GetWindowText" not in dir():
```

Comment:
Doesn't work on Windows. The only element of dir() in my testing was 'self'. If there is no way of getting name of  window with focus on Linux you can check platform instead:
import platform
if platform.system().lower() == 'linux':
    self.frames_draw_allowed = True
    return

Reply:
I will modify it on master, let me know if my change doesn't work on Linux.